finals-2024-01-16/code/5037-stefani-georgievska/src/app/models/bands.ts

```diff
@@ -0,0 +1,14 @@
+export interface Bands{
```

Comment:
+3 points for the models, the definitions for members and albums are incorrect (in typescript `[string]` and `string[]` have different meanings)